plugin/evm/atomic/vm/import_tx_test.go

```diff
@@ -1237,32 +1166,24 @@ func TestImportTxEVMStateTransfer(t *testing.T) {
 						AssetID: vm.Ctx.AVAXAssetID,
 					}},
 				}}
-				if err := tx.Sign(atomic.Codec, [][]*secp256k1.PrivateKey{{key}}); err != nil {
-					t.Fatal(err)
-				}
+				require.NoError(t, tx.Sign(atomic.Codec, [][]*secp256k1.PrivateKey{{key}}))
 				return tx
 			},
 			checkState: func(t *testing.T, vm *VM) {
 				lastAcceptedBlock := vm.LastAcceptedExtendedBlock()
 
-				sdb, err := vm.Ethereum().BlockChain().StateAt(lastAcceptedBlock.GetEthBlock().Root())
-				if err != nil {
-					t.Fatal(err)
-				}
+				statedb, err := vm.Ethereum().BlockChain().StateAt(lastAcceptedBlock.GetEthBlock().Root())
+				require.NoError(t, err)
 
-				avaxBalance := sdb.GetBalance(ethAddress)
-				if avaxBalance.Cmp(atomic.X2CRate) != 0 {
-					t.Fatalf("Expected AVAX balance to be %d, found balance: %d", *atomic.X2CRate, avaxBalance)
-				}
+				avaxBalance := statedb.GetBalance(ethAddress)
+				require.Equalf(t, 0, avaxBalance.Cmp(atomic.X2CRate), "Expected AVAX balance to be %d, found balance: %d", *atomic.X2CRate, avaxBalance)
```

Comment:
`cmp` comment again (at this point I suspect these were intentional, but still want to point out )